wpeverest/everest-forms
# Enhancement enable/disable log

### All Submissions:






### Changes proposed in this Pull Request:



Added option for enable/disable log.

### How to test the changes in this Pull Request:

1. Navigate to general setting.
2. Check enable/disable log option
3. Verify the logs on the tools tab. (Logs should not be stored when its disabled)

### Types of changes:




### Other information:




### Changelog entry

> Enhancement enable/disable log
